src/app/app.component.ts

```diff
@@ -11,9 +14,38 @@ export class AppComponent {
   cron = '* * * * *';
   periods = Object.keys(NgxCronService.PERIODS);
   allowedPeriods = Object.keys(NgxCronService.PERIODS);
-  languages = NgxCronService.LANGUAGES;
```

Comment:
Should we keep the languages in the service so it is exposed, just as the periods? `cronstrue` has this list as a  fixed list of languages that can be used, so I'm thinking it could be useful for a dev using the lib?

Reply:
Not sure it's needed. Seamed like it just adds to the repo size.

Reply:
In fact can this list be generated from cronstrue itself?